src/Cinematic/CNDecision.cs

```diff
@@ -0,0 +1,44 @@
+using System;
+using CoreMedia;
+using Foundation;
+using ObjCRuntime;
+
+#nullable enable
+
+#if !WATCH
+
+namespace Cinematic {
+
+#if NET
+	[SupportedOSPlatform ("tvos17.0")]
+	[SupportedOSPlatform ("macos14.0")]
+	[SupportedOSPlatform ("ios17.0")]
+	[SupportedOSPlatform ("maccatalyst17.0")]
+#else
+	[NoWatch, TV (17, 0), Mac (14, 0), iOS (17, 0), MacCatalyst (17, 0)]
+#endif
+	public enum CNDecisionIdentifierType {
+		Single,
+		Group,
+	}
+
+	public partial class CNDecision {
+
+		public CNDecision (CMTime time, long detectionId, bool isStrong, CNDecisionIdentifierType identifierType)
+		{
```

Comment:
I believe you should call base with NSObjectFlag.Empty here:

```suggestion
		public CNDecision (CMTime time, long detectionId, bool isStrong, CNDecisionIdentifierType identifierType)
			: base (NSObjectFlag.Empty)
		{
```